packages/firestore/scripts/prepare-test.ts

```diff
@@ -0,0 +1,67 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+prepareTest();
+
+function prepareTest() {
```

Comment:
Have you tried just updating the import paths, similar to what we do for platform injection? 

https://github.com/firebase/firebase-js-sdk/blob/master/packages/firestore/rollup.shared.js#L65

I also wonder if this new setup makes is hard to use IDEs to run the test since the won't invoke this rule.

Reply:
I don't think that it work for node tests?
For sure it's one more thing you need to do, but you only have to do it once.

I think long term, we want to rewrite these tests and remove the dependencies on the compat layer.